STYLEGUIDE.mdx

```diff
@@ -327,7 +328,7 @@ if (width < 300) {
 ### Modules
 
 Module dependencies should be written using native ES2015 syntax wherever
-possible (which is almost everywhere):
```

Comment:
This is just a nit, but I think the colons are actually helpful because it makes it clearer in cases like this that the code snippet is being provided as an example of the sentence directly preceding it.

Reply:
yup that makes sense. I've reverted this one and the others I had changed